tutorials/camkes-vm-kernel-module/module_minimal/modules/hello/hello.c

```diff
@@ -0,0 +1,30 @@
+<!--
+  Copyright 2021 Michael Neises
+
+  SPDX-License-Identifier: BSD-2-Clause
+-->

```

Comment:
Should use C comments:
```suggestion
/*
 * Copyright 2021 Michael Neises
 *
  * SPDX-License-Identifier: BSD-2-Clause
  */
```